docs/installation.md

```diff
@@ -0,0 +1,125 @@
+## Installation
```

Comment:
Should the installation section in README.md just link here? Looks like they contain mostly the same stuff